test/e2e/OWNERS

```diff
@@ -0,0 +1,16 @@
+filters:
+  # special checks that run after the entire suite
+  # must be specially reviewed
+  "e2e_invariants.go":
```

Comment:
Do we want all invariant checks in a single file? Can we use a wildcard here like `e2e_*_invariants.go` and then do `e2e_invariants.go` (common definition), `e2e_metric_invariants.go` (your code) and `e2e_log_invariants.go` (something that I intent to add, for example to check for `DATA RACE` - https://github.com/kubernetes/kubernetes/pull/133834)?

Reply:
regexp works (https://www.kubernetes.dev/docs/guide/owners/#filters ).

Reply:
Let's use a package.

Reply:
moved to a pacakge.